### DISCUSSION TITLE
EMQ X 5.0-alpha.4 Released

### DISCUSSION THREAD

**Discussion by Rory-Z** [emqx/emqx#5491] [General]
EMQ X 5.0-alpha.4 is released now, it mainly includes the following changes:

**Feature**

- Dashboard provides monitoring, management, system pages, and supports viewing node topology diagrams, etc.
- Support Redis authentication
- Authorization provides HTTP API, supports creation of ACL rules and adjustment sequence
- New gateway, adding support for CoAP
- Hot configuration provides HTTP API, supports updating configuration at runtime

<hr /><em>This discussion was created from the release <a href='https://github.com/emqx/emqx/releases/tag/v5.0-alpha.4'>EMQ X 5.0-alpha.4 Released</a>.</em>

**Comment by 1995parham** [comment]
I am really excited about v5.0. is there any docker image for it?

**Reply by Rory-Z** [reply]
Sorry, our alpha version does not push images, you can clone our code and run make emqx-docker to build the image

**Comment by Narekmat** [comment]


**Reply by Rory-Z** [reply]
Did you build the package yourself? Please describe your environment
